doc/source/release/1.13.0-notes.rst

```diff
@@ -312,6 +326,7 @@ Issues closed for 1.13.0
 * `#19774 <https://github.com/scipy/scipy/issues/19774>`__: DOC: Detail what "concatenate" means in the context of \`spatial.transform.Rotation.concatenate\`
 * `#19799 <https://github.com/scipy/scipy/issues/19799>`__: DOC: array types: update array validation guidance
 * `#19813 <https://github.com/scipy/scipy/issues/19813>`__: BUG: typo in specfun.f?
+* `#19819 <https://github.com/scipy/scipy/issues/19819>`__: BUG: In RPFInterpolator, wrong warning message if degree=-1
```

Comment:
very minor, typo: `RPFInterpolator` -> `RBFInterpolator` (I fixed the issue title itself just now)